README.md

```diff
@@ -166,35 +166,7 @@ specification:
       valid requests make it up until here. You can also specify an own function resolver function in order to hook into
       your own framework.
 
-## Development on swagger1st
```

Comment:
Sure. The current wording above is a bit ambiguous (could also be interpreted as "development status") so I'd use "Contributing to Spot On!" or something with "contributing."